spec/index.bs

```diff
@@ -703,12 +703,20 @@ This specification defines the following [=usage scopes=]:
   2. an individual command on a {{GPUComputePassEncoder}}, such as {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)|GPUProgrammablePassEncoder.setBindGroup}}.
   3. the whole {{GPURenderPassEncoder}}.
 
-Note: calling {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)|GPUProgrammablePassEncoder.setBindGroup}}
-adds the {{GPUBindGroup/[[usedResources]]}} to the [=usage scope=]
-regardless of whether the shader or {{GPUPipelineLayout}} actually depends on these bindings.
-Similarly {{GPURenderEncoderBase/setIndexBuffer()|GPURenderEncoderBase.setIndexBuffer}}
-add the index buffer to the usage scope (as [=internal usage/input=])
-regardless of whether the indexed draw calls are used afterwards.
+<div class=note>
+  Resources that are used in state setting calls are always added to current [=usage scope=] as used resources.
```

Comment:
Please use 4-space indentation for consistency with the more-recently edited parts of the document.

Reply:
Done.